go/src/koding/kites/config/konfig.go

```diff
@@ -69,35 +70,65 @@ func (e *Endpoints) Social() *Endpoint {
 	return e.Koding.WithPath("/api/social")
 }
 
-// Local describes configuration of local paths.
-type Local struct {
+// Mount describes configuration of mounts.
+type Mount struct {
 	// Mount is a default home path of mounted directories.
 	//
 	// If empty, defaults to ~/koding/mnt/.
-	MountHome string `json:"mountHome,omitempty"`
+	Home string `json:"home,omitempty"`
 
-	// Mounts maps named mounts to local paths.
+	// Exports maps named mounts to local paths.
 	//
-	// The Mounts["default"] export is used as
+	// The Exports["default"] export is used as
 	// a default one when caller does not specify
 	// a mount path.
 	//
-	// The Mounts["default"] defaults to $HOME.
-	Mounts map[string]string `json:"mounts,omitempty"`
+	// The Exports["default"] defaults to $HOME.
+	Exports map[string]string `json:"exports,omitempty"`
+
+	// Inspect configures behavior of mount inspect command.
+	Inspect *MountInspect `json:"inspect,omitempty"`
+
+	// Sync configures behavior of synchronization goroutines.
+	Sync *MountSync `json:"sync,omitempty"`
+
+	// Debug is a debug level used for logging within
+	// mounts.
+	//
+	//   <=0  - turns off debug logging
+	//   1    - turns on debug logging for syncer events
+	//   2-8  - reserved for future use
+	//   >=9  - turns on debug logging for fuse events
+	//
+	Debug int `json:"debug,omitempty"`
```

Comment:
in general, i am not in favor of using int for debug/verbose config. not objecting, just wanted to state :)